frontend/test_setup/fixtures/filters/filtersStoreThree.json

```diff
@@ -39,26 +39,11 @@
             "frequent": false,
             "inlineRenderMode": "button",
             "parametersLayoutType": "panel",
-            "debugProperties": {}
+            "debugProperties": { "sortNo": 20002 }
           }
         ]
       }
     ],
-    "filtersActive": [
-      {
-        "filterId": "default",
```

Comment:
why removing this from the test case?

Reply:
It's an active filter that does not exist in the data coming from backend, so it can't really be applied. Frontend allows that as we're treating whatever comes from the server as the source of truth